### PR TITLE
Started Report

### PR DESCRIPTION
**Pull request by IshikaM19**
I've started pulling out some key ideas from the dummy values I included in the database. Includes top 3 popular artists, top 3 selling artists, top genres, top 3 albums, top 3 tracks, total income, total sales...all for the previous day. Currently I'm using all the data as I didn't input enough into the database that has yesterday's date but the function is there it just has to be implemented. I also updated the schema.sql script so that it now includes an item table instead and also has an item_type table so we can differentiate between albums and tracks :)

### CODE REVIEW

File: pipeline/extract.py [hunk 1]
```diff
@@ -106,5 +106,6 @@ def extract_data_from_json(sales_json: dict) -> list[dict]:
     start = perf_counter()
     sales_data = load_sales_data(datetime.now())
     extracted_data = extract_data_from_json(sales_data)
+    print(extracted_data)
```
Comment (IB64) [inline pipeline/extract.py]: Perhaps this print statement is not needed anymore

File: report/report.py [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+    unique_sales = data.drop_duplicates(subset='sale_id', keep='first')
+    artist_sales = unique_sales.groupby(
+        'artist')['amount'].sum()
+    artist_sales = (artist_sales/100).head(3).reset_index()
+
+    return artist_sales
+
+
+def remove_duplicate_words(words: list) -> list:
```
Comment (IB64) [inline report/report.py]: Clever function!

File: pipeline/reset_database.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql -h $DB_IP -p $DB_PORT -U $DB_USER -c "DELETE FROM sale_event;"
+psql -h $DB_IP -p $DB_PORT -U $DB_USER -c "DELETE FROM item_genre;"
+psql -h $DB_IP -p $DB_PORT -U $DB_USER -c "DELETE FROM genre;"
+psql -h $DB_IP -p $DB_PORT -U $DB_USER -c "DELETE FROM item;"
+psql -h $DB_IP -p $DB_PORT -U $DB_USER -c "DELETE FROM item_type;"
+psql -h $DB_IP -p $DB_PORT -U $DB_USER -c "DELETE FROM artist;"
+psql -h $DB_IP -p $DB_PORT -U $DB_USER -c "DELETE FROM country;"
```
Comment (caithomas6789) [inline pipeline/reset_database.sh:9]: Good idea to have a shell script which can reset the database

File: pipeline/schema.sql [hunk 4]
```diff
@@ -48,9 +58,13 @@ CREATE TABLE sale_event(
...
     country_id SMALLINT NOT NULL,
     PRIMARY KEY (sale_id),
-    FOREIGN KEY (album_id) REFERENCES album(album_id),
+    FOREIGN KEY (item_id) REFERENCES item(item_id),
     FOREIGN KEY (country_id) REFERENCES country(country_id)
 );
+
+INSERT INTO item_type(item_type) VALUES
+    ('album'),
+    ('track');
```
Comment (caithomas6789) [inline pipeline/schema.sql:70]: I like how you seeded the item type database.

**Review by caithomas6789 — APPROVED** [review]

**Pull request merged**